doc/source/examples/similarity_explanations_20ng.ipynb

```diff
@@ -0,0 +1,1022 @@
+{
```

Comment:
<p>After this, please include a new cell to install <code>sentence_transformers</code></p><p><code>!pip install sentence_transformers</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>